docs/variables/time.md

```diff
@@ -0,0 +1,47 @@
+---
+id: time
+---
+
+# $(time)
+
+Returns the current time in a specified timezone. Shares this behavior with `$(currenttime)`.
+
+#### Parameters
+
+This variable takes **one** *required* parameter that is a **valid [Time Zone Database](https://www.iana.org/time-zones) name**, and **one** *optional* parameter that is a **string of tokens** specifying the output format, which defaults to `hh:mm:ss A (z)` if not provided.
+
+###### A list of all existing timezones can be found [**here**](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones#List), and a list of all available tokens can be found [**here**](https://momentjs.com/docs/#/displaying/format/).
```

Comment:
same here - i don't think this is a proper use of headings

Reply:
Refer to [**this**](https://github.com/fossadev/fossabot-docs/pull/20#discussion_r788770539).